ci/jenkins/jobs/macros.yaml

```diff
@@ -122,7 +123,8 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
-          ./ci/jenkins/test-vmc.sh --cluster-name "${{JOB_NAME}}-${{BUILD_NUMBER}}" --testcase '{conformance_type}' --coverage --codecov-token "${{CODECOV_TOKEN}}" --registry ${{DOCKER_REGISTRY}}
+          ./ci/jenkins/test-vmc.sh --cluster-name "$BUILD_TAG" --testcase '{conformance_type}' --coverage --codecov-token "${{CODECOV_TOKEN}}" --registry ${{DOCKER_REGISTRY}} --username "${CAPVC_USERNAME}" --password "${CAPVC_PASSWORD}" --k8s-version "v1.20.1"
+          exit $return_code
```

Comment:
projects.yaml should also be updated to add all necessary credential bindings. I can see that we added CAPVC_USERNAME and CAPVC_PASSWORD. We can check all jobs in projects.yaml, if any job uses builder-e2e or builder-conformance, we shall add the necessary credential bindings.